src/libs/dbHandler_test.go

```diff
@@ -20,7 +20,7 @@ const Unmet = "unmet expectation error: "
 // ==================== //
 
 func TestGetNetworkPolicies(t *testing.T) {
-	// prepare mock mysql
+	// prepare mock sqlite
```

Comment:
dont change existing testcase of mysql since we will support mysql as well. Add new test cases for sqlite.